latex/cinematic-color.sty

```diff
@@ -82,3 +82,7 @@
     \fi
 }
 \makeatother
+
+% URL typeface
+\def\UrlFont{\sf}
```

Comment:
[Above in the file](https://github.com/ves-tech/cinematiccolor/blob/b8ff454e3d96cd432f1b8f19a6af3f7522fe8cd0/latex/cinematic-color.sty#L26) I have defined a few custom commands, so a few consistency questions:

I made them all lower-case which we can change for `PascalCase` which I actually prefer (it was like that before) but in order to quickly see which our commands and not clash any LaTeX ones, I did add a `cc` prefix.

What do you prefer doing?

`ccachromaticnameref` --> `AchromaticNameRef`, `CCAchromaticNameRef`?
`UrlFont` --> `ccurlfont`, `CCUrlFont`

It might be good to move the command up with the others in any case.

Reply:
We will need to update the commands in the TeX4ht `cinematic-color.cfg` configuration file accordingly.